5edXHO8ImM9H7MGU4tcG6Q.jsligo

```diff
@@ -0,0 +1 @@
+type storage = int;type parameter = | ["Increment", int]| ["Decrement", int] | ["Reset"];type return_ = [list<operation>, storage];const main = (action: parameter, store: storage): return_ => {let storage = match(action, {Increment: n => store + n, Decrement: n => store - n, Reset: () => 0 }); return [list([]), storage]};
```

Comment:
Please remove and ignore this file

Reply:
Sorry I haven't seen that I pushed it, it's removed, and I updated the gitignore